### PR TITLE
Update Dockerfile

### PR DESCRIPTION
**Pull request by rasheedamir**

### CODE REVIEW

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-delivery.stakater.com:443/stakater-lab/inventory:0.0.0-PR-27-1-SNAPSHOT`

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-delivery.stakater.com:443/stakater-lab/inventory:0.0.0-PR-27-2-SNAPSHOT`

File: deploy/values-dev.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+applicationName: "inventory"
+deployment:
+  annotations:
+  podLabels:
+    app: inventory
+  volumes: {}
+  image:
+    repository: stakater/stakater-nordmart-inventory
+    tag: v0.0.3
```
Comment (rasheedamir) [inline deploy/values-dev.yaml]: @kahootali will we hard code the tag?

**Comment by stakater-bot-lab** [comment]
Tekton - Pipeline ran successfully and image docker.io/stakater/stakater-nordmart-inventory:SNAPSHOT-PR-27-c60e0127 is available

**Comment by stakater-bot-lab** [comment]
Tekton - Pipeline ran successfully and image docker.io/stakater/stakater-nordmart-inventory:SNAPSHOT-PR-27-c60e0127 is available

**Comment by stakater-bot-lab** [comment]
Tekton - Pipeline ran successfully and image `docker.io/stakater/stakater-nordmart-inventory:SNAPSHOT-PR-27-c60e0127` is available

**Comment by stakater-bot-lab** [comment]
Tekton - Pipeline ran successfully and image `docker.io/stakater/stakater-nordmart-inventory:SNAPSHOT-PR-27-c60e0127` is available

**Comment by stakater-bot-lab** [comment]
Tekton - Pipeline ran successfully and image `docker.io/stakater/stakater-nordmart-inventory:SNAPSHOT-PR-27-c60e0127` is available

**Comment by stakater-bot-lab** [comment]
Tekton - Pipeline ran successfully and image `docker.io/stakater/stakater-nordmart-inventory:SNAPSHOT-PR-27-c60e0127` is available

**Comment by stakater-bot-lab** [comment]
Tekton - Pipeline ran successfully and image `docker.io/stakater/stakater-nordmart-inventory:SNAPSHOT-PR-27-c60e0127` is available

**Comment by stakater-bot-lab** [comment]
Tekton - Pipeline ran successfully and image `docker.io/stakater/stakater-nordmart-inventory:SNAPSHOT-PR-27-c60e0127` is available

**Comment by stakater-bot-lab** [comment]
Tekton - Pipeline ran successfully and image `docker.io/stakater/stakater-nordmart-inventory:SNAPSHOT-PR-27-c60e0127` is available

**Comment by stakater-bot-lab** [comment]
Tekton - Pipeline ran successfully and image `docker.io/stakater/stakater-nordmart-inventory:SNAPSHOT-PR-27-c60e0127` is available

File: deploy/values-dev.yaml (hunk 1 above)
Comment (kahootali) [reply deploy/values-dev.yaml]: Doesn't matter, what value we set here, it will be overriden in pipeline by using `--set`

**Comment by stakater-bot-lab** [comment]
Tekton - Pipeline ran successfully and image `docker.io/stakater/stakater-nordmart-inventory:SNAPSHOT-PR-27-c60e0127` is available

**Comment by stakater-bot-lab** [comment]
Tekton - Pipeline ran successfully and image `docker.io/stakater/stakater-nordmart-inventory:SNAPSHOT-PR-27-90b9420c` is available

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-delivery.stakater.com:443/stakater-lab/inventory:0.0.0-PR-27-3-SNAPSHOT`

**Review by kahootali — APPROVED** [review]

**Pull request merged**